### DISCUSSION TITLE
AUDIBLE audio download impossible

### DISCUSSION THREAD

**Discussion by mdelgatto** [aclap-dev/video-downloadhelper#751] [Help and Q&A]
### Your question

Hello. This plug-in usually works very well, but no luck at all with audible. hope the problem gets fixed soon. Thanks.

### Addon & CoApp version and details

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://www.audible.com/webplayer?asin=B0876JV1JJ&contentDeliveryType=MultiPartBook&isSample=false&ref_=a_minerva_cloudplayer_B0876JV1JJ&overrideLph=false&initialCPLaunch=true",
  "error": {
    "details": "SideDownload error: : [in#0 @ 00000167e7b17680] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://d1jobzhhm62zby.cloudfront.net/90532201-3954-4582-8c27-ed0bbd297448.eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzc24iOiI5MDUzMjIwMS0zOTU0LTQ1ODItOGMyNy1lZDBiYmQyOTc0NDgiLCJuYmYiOjE3MTY5MjU5MzMyMzgsImV4cCI6MTcxNzAxMjMzMzIzOCwicGF0aCI6Ii9ia19hY3gwXzE5MjI2OC8xNDQ0OTUyMS9jZW5jL2cxIiwicXMiOnsic3Nfc2VjIjoyMCwiaXNzX3NlYyI6MTAsImlzYyI6MX0sImludHNpZyI6IlV2anh0SlJZcnJUZHVpMVpHVjFQbWtXcGtkQXExRkNEYmZsZGFfMkVNOVkifQ.KT20YKBETF5YdnADx4zDDtKkiUCKy79udgK15G4a4RM/base/bk_acx0_192268/14449521/cenc/g1/bk_acx0_192268_22_64.mp4?ss_sec=20&iss_sec=10&isc=1.\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "downloadable_id": "downloadable_1515180781438009",
    "error": "coapp_failure",
    "id": "coapp_failure_dbc16449-134c-4953-b800-5a1c9462f874"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Your URL takes me to a page that nags me for a login.  I am not about to register for this site.  I can't investigate this for you.
